### ISSUE TITLE
Manpages only show XML & HTML as options for "--output"

### ISSUE THREAD

**Issue by pjotrek-b** [MediaArea/MediaInfo#330]
I've just noticed that the manpages for mediainfo/mediainfo-gui show only XML and HTML as options for the "--output" parameter :cry: 

The "--help" displays the other options (JSON, EBUCore, etc).

I can add the info from the "--help" page to the manpages if you like?
(I'd copy/paste wording from Source/CLI/Help.cpp, so it's consistent)

**Comment by pjotrek-b** [comment]
btw: I've noticed the manpage text in multiple locations in the source tree: 

  * debian/mediainfo[-gui].pod
  * Project/OS/deb9.debian/mediainfo[-gui].pod
  * Project/OS/deb7.debian/mediainfo[-gui].pod

These seem to be identical :thinking: 
I assume any change should be applied to all 3 of them?

**Comment by JeromeMartinez** [comment]
Right, there are multiple locations for help, and also not in sync.
A corresponding PR would be appreciated.

**Comment by pjotrek-b** [comment]
Done:
PR #331
